types/webpack/index.d.ts

```diff
@@ -1594,6 +1594,14 @@ declare namespace webpack {
              */
             exec(code: string, filename: string): any;
 
+            /**
+             * Resolves the given request to a module, applies all configured loaders and calls
+             * back with the generated source, the sourceMap and the module instance (usually an
+             * instance of NormalModule). Use this function if you need to know the source code
+             * of another module to generate the result.
+             */
+            loadModule(request: string, callback: (err: Error, source: string, sourceMap: RawSourceMap, module: Module) => void): any;
```

Comment:
I'm not sure why you would call loadModule if you're not going to listen to the result? And looking at the code there are no checks for if the callback is not passed.

The arguments are optional to the callee anyway because you can omit it from your callback and it will not error. So this will satisfy those typings
```
this.loadModule('someString', (err, source) => {});
```

With regards to the caller as far as I can tell from the code (please correct me if I'm wrong) the callback always calls with all four arguments.
`line 88 webpack/lib/dependencies/LoaderPlugin.js`
```
return callback(null, source, map, dep.module);
```

I will update to type Error as Error | null as it is nullable.